chains/v20/chains_dev.json

```diff
@@ -11047,6 +11047,15 @@
                     "type": "subquery",
                     "url": "https://subquery-governance-testnet-stg.novasama-tech.org"
                 }
+            ],
+            "referendum-summary": [
+                {
+                    "type": "polkassembly",
+                    "url": "https://novasama.polkassembly.io/api/v1/posts/summary",
```

Comment:
would it better to use polkadot\kusama instead of broken ui?

Reply:
we need to test swapgov in terms of refs lifecycle, better just not to make API work as it's not working anyway to not to mess with it